engine/pong.h

```diff
@@ -38,6 +39,10 @@ class Pong
 
     static GameObject* FindGameObjectByName(const std::string& name);
 
+    static Text* AddText(const std::string& text, const std::string& fontPath,
```

Comment:
This is for creating a text object on the fly instead of in PongInit(). These parameters are probably temporary until I figure out everything that Text needs to know when created. I think it also might be a good idea to do something like emplace_back for this with GameObjects or Text. That way the unique_ptr can be created for the vector, and a reference can still be passed back.